src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessor.java

```diff
@@ -88,9 +101,17 @@ private void processBlob(String blobName, String containerName) {
                 boolean validSignature = signatureVerifier.verifyZipSignature(blobName, rawBlob);
 
                 if (validSignature) {
-                    // target storage account will be retrieved from configuration
-                    // when Crime blob processing is supported
-                    dispatcher.dispatch(blobName, rawBlob, containerName, TargetStorageAccount.BULKSCAN);
+                    ServiceConfiguration.StorageConfig containerConfig = storageConfig.get(containerName);
```

Comment:
maybe we can import this class and just use `StorageConfig`

Reply:
It's all we use from that config. I think it's a bit less polluting to retrieve that important bit in the constructor and refer to it from then on, than adding that part to blob-processing logic.

Reply:
I mean replacing `ServiceConfiguration.StorageConfig` with `StorageConfig`

Reply:
Aaa, sorry, ok. It's internal, but I'll move it out.

Reply:
even better

Reply:
Would you be ok with me doing it in another PR? I'll rename the `name` property there, as well.

Reply:
ok